content/en/Platform Deep Dive/Cobalt Account/account-recovery.md

```diff
@@ -86,6 +88,33 @@ To reset your password:
 1. Enter your email address that you used to [sign in to Cobalt](/getting-started/sign-in/), and select **Reset Password**.
 1. Follow the instructions in the email you receive. Learn [how to create a strong password](/platform-deep-dive/cobalt-account/password-best-practices/).
 
+## Your Account Isn't Fully Set Up
+
+To use the Cobalt platform, you need to fully set up your account.
+
+1. Ask your [Organization Owner](/platform-deep-dive/collaboration/user-roles/#organization-owner) to [invite you to your organization](/platform-deep-dive/organization/manage-users/#invite-users) in Cobalt, if they haven't already done so.
+1. Confirm your email address by clicking the link in the email invitation.
+1. Create a password.
+
+Possible problems:
+
+- If you're not receiving emails from Cobalt, see our [troubleshooting tips](/platform-deep-dive/collaboration/manage-notifications/#troubleshoot-email-notifications).
+- If the invitation link in the email has expired, ask your Organization Owner to invite you to the organization once again.
+  - If you're an Organization Owner: To resend an invitation to a user, [remove them from your organization](/platform-deep-dive/organization/manage-users/#remove-users), and then [invite](/platform-deep-dive/organization/manage-users/#invite-users) them again.
+  - If you're the only Organization Owner in your organization: To get help, contact your Customer Success Manager (CSM) or support@cobalt.io.
+
+## Your Email or Password is Invalid
+
+When you sign in, you may see an error stating that your email or password is invalid.
+
+Ensure that:
+
+- You're using the correct email address that you registered with.
```

Comment:
Do we need to be super clear here? And say something like Ensure that you're email has been invited?

Reply:
Updated this.